packages/core/src/storage.ts

```diff
@@ -1,50 +1,36 @@
 import type { Readable } from "stream";
 import { isReadableStream } from "./utils";
+import {
+  StorageRequestStatus,
+  getSdk,
+  StatusByIdQuery,
+  StatusByCidQuery,
+} from "./client";
+import { GraphQLClient } from "graphql-request";
 
-/**
- * Status is the status of a StorageRequest.
- * * `Unknown` is the default value to an unitialized StorageRequest. This status must be
- *   considered invalid in any real StorageRequest instance.
- * * `Batching` indicates that the storage request is being batched.
- * * `Preparing` indicates that the batch containing the data is being prepared.
- * * `Auctioning` indicates that the batch containing the data is being auctioned.
- * * `DealMaking` indicates that the data is in deal-making process.
- * * `Success` indicates that the request was stored in Filecoin.
- * * `Error indicates that there is some error handling the request.
- */
-type Status =
```

Comment:
dropping this in favor of the enum we get from the client builder, much easier to keep in sync!